packages/core/docs/performance/other-optimizations.md

```diff
@@ -0,0 +1,123 @@
+
+## Avoid render-blocking resources :orange_book:
+
+Render-blocking resources are scripts, stylesheets, and other imports in the `<head>` that delay the browser from rendering page content to the screen until they are downloaded and parsed. Such resources delay the First Paint - time needed for the browser to render something (i.e., background colors, borders, text, or images) for the first time. To prevent that:
+
+* add the `defer` or `async` attribute to the `<script>`,
+* use `disabled` or `media` attributes for the stylesheets.
+
+Nuxt is deferring every bundle script it generates. If you want to deffer third party scripts, just add them in the `nuxt.config` as shown in the example below:
+
+```javascript
+// nuxt.config.js
+
+export default {
+  head: {
+    script: [
+      {
+        src: `<SCRIPT_URL>`,
+        defer: true
+      }
+    ]
+  }
+};
+```
+
+Additionally, you can add use [media-dependent](https://developer.mozilla.org/en-US/docs/Web/CSS/@import) import in your CSS:
+
+```css
+//main.scss
+@import '@/assets/main.scss' print;
+```
+
+## Inline critical assets :ledger:
+
+Consider delivering critical JavaScript/CSS inline and deferring all non-critical assets. You can reduce the size of your pages by only shipping the code and styles that you need at the given time.
+
+Once you've identified critical code, move that code from the render-blocking URL to an inline `<script>` tag. Do the same for the styles required for the first paint inside a `<style>` block at the `<head>` of the HTML page and load the rest of the styles asynchronously using the `preload` link.
+
+## Swap fonts :ledger:
+
+Leverage the `font-display` CSS feature to ensure that the text is visible to the user while web fonts are loading.
+
+```css
+@font-face {
+  font-family: 'Arial';
+  font-display: swap;
+}
+```
+
+The `font-display` API specifies how a font is displayed. The `swap` value tells the browser that the text using this font should be displayed immediately using a system font. Once the custom font is ready, it replaces the system font.
+
+If you are using Google Fonts, add the `&display=swap` parameter to the end to the Google Fonts URL:
+
+```html
+<link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
+```
+
+If you are using the `@nuxtjs/google-fonts` package, you can use the [display](https://google-fonts.nuxtjs.org/options#display) property.
+
+## Avoid using more than one library version per page :ledger:
+
+Check if your website doesn't use multiple versions of the same library. This will make the user download unnecessary data and slow down your page. Clean up the code and make sure you only use one version by running the following command. It will generate a report of what's in the final bundle.
+
+```bash
+yarn nuxt build --analyze
+```
+
+## Compress text content :ledger:
```

Comment:
```suggestion
## Compress your files :ledger:
```

We don't need to be too specific if it could be misleading to people who don't know much about perf

Reply:
Fixed